### DISCUSSION TITLE
NEED TO CHANGE THE SIGNATURE ON THE EXISTING CERTIFICATE.

### DISCUSSION THREAD

**Discussion by BimleshTrigyn** [Sunbird-Lern/Community#142] [Issues] [answered]
Hi Team,

We got,  one requirement where, we need to change the signature on all the existing certificate. In a particular given tenant/organization.
Can you provide some input here , how to approach and  solve it..?.

If possible, please provide The API details.

Thanks.

**Comment by AmiableAnil** [comment, marked as answer]
@BimleshTrigyn Please refer here https://lern.sunbird.org/learn/product-and-developer-guide/batch-service/certificate-flow
Please let us know if this resolves your issue.

**Reply by AmiableAnil** [reply]
@BimleshTrigyn Please let us know if this issue is resolved or not. Or do you need further some help on this?

**Reply by BimleshTrigyn** [reply]
Hii @AmiableAnil , thanks, for replay, yes the issue got resolved.

Thanks.
